server/src/test/java/org/elasticsearch/index/mapper/flattened/KeyedFlattenedFieldTypeTests.java

```diff
@@ -31,7 +31,7 @@
 public class KeyedFlattenedFieldTypeTests extends FieldTypeTestCase {
```

Comment:
It'd be nice to add a unit test here -- we've been adding tests like `RootFlattenedFieldTypeTests#testFetchSourceValue` for all the field type that support fetching.